docs/installation/docker-compose.md

```diff
@@ -0,0 +1,299 @@
+14. Build the autograding image(s) that you want to use in Tango (see [the docs](https://docs.autolabproject.com/tango-vmms/#docker-vmms-setup) for more information). For this setup we will stick to the default Ubuntu 18.04 autograding image: 
+
+        :::bash
+        docker build -t autograding_image Tango/vmms/
+        
+Note that we can just run this directly on the host because we are mapping the Docker socket to the Tango container (i.e they are using the same Docker server).
+
+15. Start up everything: 
+
+        :::bash
+        docker-compose up -d
+        
+Autolab should now be accessible on port 80 (and 443 if you configured TLS)!
+
+## Configuring SSL/TLS
+There are three options for TLS: using Let's Encrypt (for free TLS certificates), using your own certificate, and not using TLS (not recommended for production deployment).
+
+### Option 1: Let's Encrypt
+1. Ensure that your DNS record points towards the IP address of your server
+2. Ensure that port 443 is exposed on your server (i.e checking your firewall, AWS security group settings, etc)
+3.  Get initial SSL setup script: `make ssl`
+4. In `ssl/init-letsencrypt.sh`, change `domains=(example.com)` to the list of domains that your host is associated with, and change `email` to be your email address so that Let's Encrypt will be able to email you when your certificate is about to expire
+5. If necessary, change `staging=0` to `staging=1` to avoid being rate-limited by Let's Encrypt since there is a limit of 20 certificates/week. Setting this is helpful if you have an experimental setup.
+6. Run your modified script: `sudo sh ./ssl/init-letsencrypt.sh`
+
+### Option 2: Using your own TLS certificate
+1. Copy your private key to `ssl/privkey.pem`
+2. Copy your certificate to `ssl/fullchain.pem`
+3. Generate your dhparams:
+
+        :::bash
+        openssl dhparam -out ssl/ssl-dhparams.pem 4096
+
+4. Uncomment the following lines in `docker-compose.yml`:
+
+        :::bash
+        # - ./ssl/fullchain.pem:/etc/letsencrypt/live/test.autolab.io/fullchain.pem;
+        # - ./ssl/privkey.pem:/etc/letsencrypt/live/test.autolab.io/privkey.pem;
+        # - ./ssl/ssl-dhparams.pem:/etc/letsencrypt/ssl-dhparams.pem
+
+### Option 3: No TLS (not recommended, only for local development/testing)
+1. In `docker-compose.yml` (for all the subsequent steps), comment out the following:
+
+        :::bash
+        # Comment the below out to disable SSL (not recommended)
+        - ./nginx/app.conf:/etc/nginx/sites-enabled/webapp.conf
+    
+    Also uncomment the following:
+
+        :::bash
+        # Uncomment the below to disable SSL (not recommended)
+        # - ./nginx/no-ssl-app.conf:/etc/nginx/sites-enabled/webapp.conf
+    
+    Lastly set `DOCKER_SSL=false`:
+
+        :::bash
+        environment:
+        - DOCKER_SSL=true                         # set to false for no SSL (not recommended)
+
+## Updating Autolab/Tango Deployment
```

Comment:
Perhaps call it updating docker compose deployment instead?

Reply:
Updated